fury/ui.py

```diff
@@ -1100,11 +1100,18 @@ def __init__(self, text="Text Block", font_size=18, font_family='Arial',
             Makes text italicised.
         shadow : bool
             Adds text shadow.
+        size : (int, int)
+            Size (width, height) in pixels of the Text.
         """
         super(TextBlock2D, self).__init__(position=position)
         self.color = color
         self.background_color = bg_color
-        self.font_size = font_size
+        if font_size is not None:
+            self.font_size = font_size
+        if size is not None:
+            self.size = size
+        if font_size is None and size is None:
+            self.font_size = 18
```

Comment:
I do not understand why you need to do all of that. You could add default value in the `__init__`. Can you explain your chaoice ? thank you

Reply:
Actually, the problem is when I set the `position2` coordinate, the actor ignores the font parameter. Therefore when I set the default size to `(100, 100)`, the size of the actor remains the same(100, 100) no matter how big the font size is.

Reply:
I was testing if such if statements would help or not but it didnt